src/main/java/genji/Parser.java

```diff
@@ -0,0 +1,120 @@
+package genji;
+
+import genji.command.Command;
+import genji.command.MarkCommand;
+import genji.command.UnmarkCommand;
+import genji.command.AddCommand;
+import genji.command.DeleteCommand;
+import genji.command.DateCommand;
+import genji.command.ExitCommand;
+import genji.command.ListCommand;
+import genji.task.ToDo;
+import genji.task.Deadline;
+import genji.task.Event;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeParseException;
+
+public class Parser {
+    public static Command parse(String command) throws GenjiException{
```

Comment:
Similar as before add a whitespace between the `GenjiException`  and the  { to follow the coding standard.